src/components/chat/tool-icon-component.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { useContext } from "react";
+import { useTypeOfTileInDocumentOrCurriculum } from "../../hooks/use-stores";
+import { AppConfigContext } from "../../app-config-context";
```

Comment:
```suggestion
import { AppConfigContext } from "../../app-config-context";
import DocumentIcon from "../../assets/icons/document-icon.svg";
```
utilizing the new icon I added to the PR in a commit.

Reply:
Done.